code/datums/api.dm

```diff
@@ -633,10 +633,48 @@ proc/api_update_command_database()
 		data = null
 		return 1
 
+//Get Server Status
+/datum/topic_command/get_serverstatus_public
+	name = "get_serverstatus_public"
+	description = "Gets the serverstatus"
+	no_auth = 1
+
+/datum/topic_command/get_serverstatus/run_command(queryparams)
+	var/list/s[] = list()
+	s["version"] = game_version
+	s["mode"] = master_mode
+	s["respawn"] = config.abandon_allowed
+	s["enter"] = config.enter_allowed
+	s["vote"] = config.allow_vote_mode
+	s["ai"] = config.allow_ai
+	s["host"] = host ? host : null
+	s["players"] = 0
+	s["admins"] = 0
+	s["stationtime"] = worldtime2text()
+	s["roundduration"] = get_round_duration_formatted()
+	s["gameid"] = ""
```

Comment:
Is it intended that this is always blank?

Reply:
Intended to make this information actually not provided. Probably it's other thing if bot's can't handle missing values.